test/gie/spilhaus.gie

```diff
@@ -0,0 +1,3436 @@
+<gie-strict>
+
```

Comment:
That's a lot of test points. Do we need so much ? Especially given there is hardly any branch in the implementation. Perhaps keep a few ones close to the projection center and ones far away that are challenging for numerical precision ?

Reply:
During the development I spent a lot of time until I realized that longitudes should be normalized before calling Adams_ws2. It failed only aprox in one eighth of the projection. That made me a bit paranoid about sampling more points.
The first case is every 30 deg. That is not that much... or maybe yes?
The second group is getting every 100th element of the file from ersi (and removing 2 outlayers).
Runtime is 0.02 s (in debug).

Reply:
I don't have a too strong opinion about this. Probably more a personal taste where I tend to keep testing at the strict minimum to reduce their potential future maintenance.

Reply:
I reduced significantly the sample points. In the generated points, the spacing is bigger. For the samples from ESRI, I picked manually (with the help of a geojson) some points around the projection.